frequency-chain/frequency-xyz
# update deps and fix warnings

# Description
Updates a bunch of packages including Svelte 5, and other major versions.  Fixes some warnings.

- Closes #28 

## Type of change
- [x] CI/Dependencies/etc...

# How to Test?
Click on the latest preview deployment link, below.
1. Verify it runs/doesn't crash, doesn't have warnings
2. Click through the numerical slides and verify they all work
3. Click the right side tab and make sure that shows up
4. click around elsewhere

# Checklist:
